src/systemtask/SystemTask.cpp

```diff
@@ -181,7 +181,7 @@ void SystemTask::Work() {
     UpdateMotion();
 
     Messages msg;
-    if (xQueueReceive(systemTasksMsgQueue, &msg, 100) == pdTRUE) {
```

Comment:
How come there was no need to convert the 100ms to ticks before, but now we do?

Reply:
`configTICK_RATE_HZ` is set to 1024, so 1 millisecond is 1024/1000 ticks=1.024 ticks
So previously it was set to 97.66ms rather than 100ms.
i.e the existing ratio is close enough to 1:1 that it wasn't noticeable, but it should be using the conversion for correctness

Reply:
Ah I see, thanks for the explanation.